docs/breaking-changes/v3.md

```diff
@@ -0,0 +1,17 @@
+# Breaking changes in v3
+
+In v3, this action no longer uses custom scripts to install tools. Instead, it
+now relies on official GitHub Setup Actions such as:
+
+-   actions/setup-node
+-   actions/setup-python
+-   actions/setup-java
+-   hashicorp/setup-terraform
+
+❗ What’s Changed
+
+-   ❌ Removed: Custom installers and tool management logic (e.g. sdkman, tfenv,
+    nodenv, pyenv).
+-   ✅ Added: Use of native GitHub-maintained setup actions.
+-   💡 Impact: If your workflow depended on .sdkmanrc, .nvmrc, or similar files
```

Comment:
This might be confusing. This change doesn't require any change in the consumer, as the api is the same right? You could instead say that this is being triggered as a breaking change to allow consumers to be able to test this change more explicitly since the internal implementation is changing a lot